### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by LenGillespie**
# Conflicts:
-	config/startup.sh
-	main.tf
-	variables.tf

These changes are mainly where I need a second pair of eyes. 

FIY - didn't go through any other file changes that pulled from upstream

### CODE REVIEW

**Comment by LenGillespie** [comment]
Forgot to run `tf validate` first. Busy fixing bugs

**Comment by LenGillespie** [comment]
`tf validate -var region=us-east1` passed.

**Review by avestalea — APPROVED** [review]
LGTM but only thing is I cannot really tell if ssh will be reachable with the new nat boxes.  I prefer they not be.  TY

File: variables.tf [hunk 1]
```diff
@@ -74,68 +94,110 @@ variable squid_config {
...
+}
+
 variable region_params {
   description = "Map of default zones and IPs for each region. Can be overridden using the `zone` and `ip` variables."
   type        = "map"
 
   default = {
-    us-west1 {
-      zone = "us-west1-b"
+    asia-east1 {
```
Comment (ChristianWitts) [inline variables.tf:130]: The `=` assignment operator is conspicuously missing from this one. I see it's the same in upstream though, but I would say it should be there.

**Review by ChristianWitts — APPROVED** [review]
Apart from the minor issue with the `region_params` default map, looks good 👍 🚀

**Pull request merged**